src/components/Leaderboard/Leaderboard.module.css

```diff
@@ -1,34 +1,54 @@
 .leaderboard {
-  margin-bottom: 10em;
-  position: relative;
   overflow: hidden;
-  padding: 10px;
+  padding: 10px 10px 25px 10px;
   right: 20%;
-  border-radius: 35px;
+  border-radius: 35px !important;
   border: 4px solid #fafafa;
-  background: rgba(250, 250, 250, 0.4);
+  background-color: rgba(11, 22, 102, 0.6) !important;
```

Comment:
All those colors will be change by globalStyle,  dont worry about ... i will create ticket to replace all

Reply:
ok